src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -14,858 +14,1080 @@ namespace System.Runtime.Intrinsics.X86
         public static Vector128<short> Or(Vector128<short> left, Vector128<short> right) => Or(left, right);
         /// <summary>
-        /// __m128i _mm_or_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_or_si128 (__m128i a,  __m128i b);
+        ///   POR xmm, xmm/m128
         /// </summary>
         public static Vector128<ushort> Or(Vector128<ushort> left, Vector128<ushort> right) => Or(left, right);
         /// <summary>
-        /// __m128i _mm_or_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_or_si128 (__m128i a,  __m128i b);
+        ///   POR xmm, xmm/m128
         /// </summary>
         public static Vector128<int> Or(Vector128<int> left, Vector128<int> right) => Or(left, right);
         /// <summary>
-        /// __m128i _mm_or_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_or_si128 (__m128i a,  __m128i b);
+        ///   POR xmm, xmm/m128
         /// </summary>
         public static Vector128<uint> Or(Vector128<uint> left, Vector128<uint> right) => Or(left, right);
         /// <summary>
-        /// __m128i _mm_or_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_or_si128 (__m128i a,  __m128i b);
+        ///   POR xmm, xmm/m128
         /// </summary>
         public static Vector128<long> Or(Vector128<long> left, Vector128<long> right) => Or(left, right);
         /// <summary>
-        /// __m128i _mm_or_si128 (__m128i a,  __m128i b)
+        /// __m128i _mm_or_si128 (__m128i a,  __m128i b);
+        ///   POR xmm, xmm/m128
         /// </summary>
         public static Vector128<ulong> Or(Vector128<ulong> left, Vector128<ulong> right) => Or(left, right);
         /// <summary>
-        /// __m128d _mm_or_pd (__m128d a,  __m128d b)
+        /// __m128d _mm_or_pd (__m128d a,  __m128d b);
+        ///   ORPD xmm, xmm/m128
         /// </summary>
         public static Vector128<double> Or(Vector128<double> left, Vector128<double> right) => Or(left, right);
 
         /// <summary>
-        /// __m128i _mm_packs_epi16 (__m128i a,  __m128i b)
+        /// __m128i _mm_packs_epi16 (__m128i a,  __m128i b);
+        ///   PACKSSWB xmm, xmm/m128
         /// </summary>
         public static Vector128<sbyte> PackSignedSaturate(Vector128<short> left, Vector128<short> right) => PackSignedSaturate(left, right);
         /// <summary>
-        /// __m128i _mm_packs_epi32 (__m128i a,  __m128i b)
+        /// __m128i _mm_packs_epi32 (__m128i a,  __m128i b);
+        ///   PACKSSDW xmm, xmm/m128
         /// </summary>
         public static Vector128<short> PackSignedSaturate(Vector128<int> left, Vector128<int> right) => PackSignedSaturate(left, right);
 
         /// <summary>
-        /// __m128i _mm_packus_epi16 (__m128i a,  __m128i b)
+        /// __m128i _mm_packus_epi16 (__m128i a,  __m128i b);
+        ///   PACKUSWB xmm, xmm/m128
         /// </summary>
         public static Vector128<byte> PackUnsignedSaturate(Vector128<short> left, Vector128<short> right) => PackUnsignedSaturate(left, right);
 
```

Comment:
It seems that `void _mm_pause (void)` or `PAUSE` intrinsic is missing

Reply:
Explicitly avoiding this one as it could have undesired impacts on threading/GC, etc.

Reply:
Needs a separate, in depth discussion, to determine if it is okay to expose.

Reply:
pause is already exposed as `Thread.SpinWait()`. It is not inlined by the JIT today, but that can be fixed... .

It is incredibly hard to use this instruction directly because of it has very different performance characteristics on different process (https://github.com/dotnet/coreclr/issues/13388).

Reply:
@tannergooding @fiigii - `void _mm_pause (void)` presence is not controlled by any CPUID flag and is available since Pentium 4 there is no reson to exclude it - anyway I remember that in corefx repo there were several issues with correctly tuning spin locks and this instruction could be a perfect solution to this problems

Reply:
You have to always wrap this instruction into a construct like `Thread.SpinWait()` that gives you some consistency to get your spin loop tuned properly. I do not exposing this instruction directly would help with anything.

Reply:
@jkotas, if it is already "exposed" and shouldn't cause issues with threading or the GC, would it be good to expose here anyways, for completeness?

Reply:
I do not think so.

Reply:
I think in such case problem is already "solved" by instruction being hard to use

Reply:
Seems that github is working today really very slow and I got out of the context very fast

Reply:
> Thread.SpinWait() that gives you some consistency to get your spin loop tuned properly
> has very different performance characteristics on different processor models 

Right, we already did lots of work to tune the spin.